### PR TITLE
Add missing spanish translations

### PR DESCRIPTION
**Pull request by Senen**
## References

* Pull Requests: #3500
* Commit e06bbaed missed these translations

## Objectives

Add spanish missing translations.

Running `i18n-tasks` for `es` locale i realized some translations were missing. After this change both `i18n-tasks` (unused and missing) are passing without errors.

### CODE REVIEW

File: config/locales/es/general.yml [hunk 1]
```diff
@@ -682,6 +682,8 @@ es:
       multiple: "Puedes seleccionar un máximo de %{maximum} respuestas"
       answer_couples_open: "Elige una opción entre el siguiente par. Puedes elegir un máximo de %{maximum} respuestas"
       answer_couples_closed: "Elige una opción entre el siguiente par. Puedes elegir un máximo de %{maximum} respuestas"
+      answer_set_open: "Puede elegir un máximo de %{máximo} respuestas."
```
Comment (javierm) [inline config/locales/es/general.yml]: The interpolation variable should not be translated :wink:.

File: config/locales/es/general.yml [hunk 2]
```diff
@@ -682,6 +682,8 @@ es:
...
+      answer_set_closed: "Puede elegir un máximo de %{máximo} respuestas."
```
Comment (javierm) [inline config/locales/es/general.yml]: The interpolation variable should not be translated :wink:.

File: config/locales/es/activemodel.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ es:
       verification:
         residence: "Residencia"
         sms: "SMS"
-      local_census_records/import:
+      local_census_record/import:
```
Comment (javierm) [inline config/locales/es/activemodel.yml]: Actually, I think this key is OK, and it's the `config/locales/en/activemodel.yml` file which needs to be modified, according to the console on the master branch:

```
> I18n.locale = :en
=> :en
> LocalCensusRecords::Import.model_name.human
=> "Import"
> I18n.locale = :es
=> :es
> LocalCensusRecords::Import.model_name.human
=> "Importación de registros del censo local"
```

File: config/locales/es/activemodel.yml [hunk 4]
```diff
@@ -36,7 +36,7 @@ es:
         document_type: "Tipo de documento"
         document_number: "Número de documento (incluida letra)"
         year_of_birth: "Año de nacimiento"
-      local_census_records/import:
+      local_census_record/import:
```
Comment (javierm) [inline config/locales/es/activemodel.yml]: Actually, I think this key is OK, and it's the `config/locales/en/activemodel.yml` file which needs to be modified, according to the console on the master branch:

```
> I18n.locale = :en
=> :en
> LocalCensusRecords::Import.human_attribute_name(:file)
=> "File"
> I18n.locale = :es
=> :es
> LocalCensusRecords::Import.human_attribute_name(:file)
=> "Archivo"
```

**Comment by Senen** [comment]
Haste makes waste! 🤦‍♂ 🙏 @javierm

**Pull request merged**